.github/workflows/test_adaptersv2.yaml

```diff
@@ -0,0 +1,258 @@
+# This workflow will be called/referenced in each of the adapters and runs in the context of adapter
+name: Adapter Integration - Core v2
+
+on:
+  workflow_call:
+    inputs:
+      expected_resources:
+        required: true
+        type: string
+      expected_resources_types:
+        required: true
+        type: string
+      expected_resources_namespaces:
+        required: true
+        type: string
+      service_url:
+        required: false
+        type: string
+      deployment_url:
+        required: false
+        type: string
+      adapter_name:
+        required: false
+        type: string
+        default: meshery
+      patternfile_name:
+        required: true
+        type: string
+      k8s_cni:
+        required: false
+        type: string
+        default: 'bridge'
+      k8s_version:
+        required: false
+        type: string
+      sm_version:
+        required: false
+        type: string
+      output_filename:
+        required: true
+        type: string
+      adapter_version:
+        required: false
+        type: string
+      provider: #default to local
+        required: false
+        type: string
+    secrets:
+      token:
+        required: true
+
+jobs:
+  TestAdapter:
+    strategy:
+      fail-fast: false
+      matrix:
+        k8s_version: ['v1.20.11', 'v1.21.5', 'v1.22.2','v1.23.9']
```

Comment:
No, 1.24 or 1.25?

Reply:
For now, due to problems in other adapters. We can keep this to 1.23.

Reply:
@Revolyssup the issue here is that minikube and docker-desktop are both already running v1.25. We need to get to 1.25 now. What's preventing us from moving forward?